models/model.js

```diff
@@ -7,4 +7,17 @@ function fetchTopics() {
     
 }
 
-module.exports = {fetchTopics}
\ No newline at end of file
+function fetchArticleId(article_id) {
+    return db.query(`SELECT * FROM articles WHERE article_id=$1;`, [article_id])
+    .then(({rows: articles}) => {
+        if(!articles[0]) {
+            return Promise.reject({status: 404, msg: "article_id not found"})
+        }
+        return articles
```

Comment:
The aim of this function is to fetch a single article, so it would make sense if this were 
- not an array, but the first value of the array
- called `article` rather than `articles`